src/libaktualizr-c/test/CMakeLists.txt

```
@@ -5,6 +5,8 @@ SET(CMAKE_SKIP_RPATH TRUE)
 
 add_executable(${TARGET_NAME} EXCLUDE_FROM_ALL ${SOURCES})
 add_dependencies(build_tests ${TARGET_NAME})
-target_link_libraries(${TARGET_NAME} aktualizr-c)
+target_link_libraries(${TARGET_NAME} aktualizr-c api-test-utils)
 
 aktualizr_source_file_checks(${SOURCES})
+
+add_subdirectory(api-test-utils)
```

To remove these tests from the default target:

```suggestion
add_subdirectory(api-test-utils EXCLUDE_FROM_ALL)
```

---

Fixed.